pages/docs/modules/exec.mdx

```diff
@@ -0,0 +1,355 @@
+>>> cmd.run()
+>>> cmd.stdout
+"1234\n"
+```
+
+### command.stdout
+
+```go filename="Field signature"
+command.stdout io.Writer
+```
+
+The process' standard output pipe. Defaults to a buffer.
+
+If read after process has been executed, and the value was left as default,
+then the field will return a string.
+
+```go copy filename="Example"
+>>> cmd := exec.command("bash", "-c", "echo hello stdout; echo hello stderr >&2")
+>>> cmd.run()
+>>> cmd.stdout
+"hello stdout\n"
+```
+
+### command.stderr
+
+```go filename="Field signature"
+command.stderr io.Writer
+```
+
+The process' standard error pipe. Defaults to a buffer.
+
+If read after process has been executed, and the value was left as default,
+then the field will return a string.
+
+```go copy filename="Example"
+>>> cmd := exec.command("bash", "-c", "echo hello stdout; echo hello stderr >&2")
+>>> cmd.run()
+>>> cmd.stderr
+"hello stderr\n"
+```
+
+### command.run
+
+```go filename="Method signature"
+command.run()
+```
+
+Run starts the command and waits for it to complete.
+Run will error if called multiple times.
+
+The result can be found in the command object's other fields, such as
+[`command.stdout`](#commandstdout),
+when the process exists with a zero exit code.
+
+To capture the output in the case of a non-zero exit code, use [`try`](../builtins.mdx#try).
+
+```go copy filename="Example"
+>>> cmd := exec.command("bash", "-c", "exit 0")
+>>> cmd.run()
+>>> cmd2 := exec.command("bash", "-c", "exit 1")
+>>> cmd2.run()
+exit status 1
+>>> cmd3 := exec.command("bash", "-c", "echo message from failed app; exit 1")
+>>> try(cmd3.run, "uh oh")
+"uh oh"
+>>> cmd3.stdout
+"message from failed app\n"
+```
+
+### command.combined_output
+
+```go filename="Method signature"
+command.combined_output() byte_slice
+```
+
+Similar to [`command.run`](#commandrun), but returns the combined output
+of the process' both standard output and standard error as a byte slice.
+
+```go copy filename="Example"
+>>> cmd := exec.command("bash", "-c", "echo hello stdout; echo hello stderr >&2")
+>>> cmd.combined_output()
+byte_slice("hello stdout\nhello stderr\n")
+```
+
+### command.output
+
+```go filename="Method signature"
+command.output() byte_slice
+```
+
+Similar to [`command.run`](#commandrun), but returns the process'
+standard output as a byte slice.
+
+```go copy filename="Example"
+>>> cmd := exec.command("bash", "-c", "echo hello stdout; echo hello stderr >&2")
+>>> cmd.output()
+byte_slice("hello stdout\n")
+```
+
+### command.environ
+
+```go filename="Method signature"
+command.environ() list
+```
+
+Returns a copy of the environment in which the command would be run as it is
+currently configured.
+
+```go copy filename="Example"
+>>> cmd := exec.command("bash", "-c", "echo hello stdout; echo hello stderr >&2")
+>>> cmd.environ()
+["TERM=xterm-256color", "SHELL=/bin/bash", "USER=alice", ...]
+```
+
+### command.start
+
+```go filename="Method signature"
+command.start()
+```
```

Comment:
Another design idea: Would be nice if `start` and `wait` returned a boolean instead, where a `true` means they successfully started/waited, but `false` if the process was already started/waited.